src/System.IO.FileSystem/tests/FileStream/Dispose.cs

```diff
@@ -58,6 +64,94 @@ protected override void Dispose(bool disposing)
             }
         }
 
+
+        [Fact]
+        public void Dispose_ThrowDuringFlushWriteBuffer_TrueArgThrows()
+        {
+            RemoteInvoke(() =>
+            {
+                string fileName = GetTestFilePath();
+                using (FileStream fscreate = new FileStream(fileName, FileMode.Create))
+                {
+                    fscreate.WriteByte(0);
+                }
+                bool writeDisposeInvoked = false;
+                Action<bool> writeDisposeMethod = (disposing) =>
```

Comment:
Nit, I would write in one line
`Action<bool> writeDisposeMethod = (disposing =>writeDisposeInvoked = true; };`

Reply:
Or even more concisely:
```C#
Action<bool> writeDisposeMethod = _ => writeDisposeInvoked = true;
```

Reply:
I made this change on line 79. I left this as is because the method is not a one-liner, it also asserts disposing is expected to call with false argument.